.travis.yml

```diff
@@ -20,19 +21,13 @@ gemfile:
 
 env:
   global:
-    - QMAKE=/usr/lib/x86_64-linux-gnu/qt5/bin/qmake SQLITE=memory DBCLEANER=deletion
+    - QMAKE=/usr/lib/x86_64-linux-gnu/qt5/bin/qmake
```

Comment:
NB: the SQLITE=memory actually does nothing (should be DATABASE_FILE=:memory: if you wanted to switch sqlite to memory)